packages/passport/sdk/src/starkEx/workflows/order.ts

```diff
@@ -33,66 +32,64 @@ export async function createOrder({
   ordersApi,
   guardianClient,
 }: CreateOrderParams): Promise<imx.CreateOrderResponse> {
-  return withPassportError<imx.CreateOrderResponse>(guardianClient.withDefaultConfirmationScreenTask(async () => {
-    const { ethAddress } = user.imx;
-    const amountSell = request.sell.type === ERC721 ? '1' : request.sell.amount;
-    const amountBuy = request.buy.type === ERC721 ? '1' : request.buy.amount;
-    const headers = { Authorization: `Bearer ${user.accessToken}` };
-
-    const getSignableOrderRequestV3: imx.GetSignableOrderRequest = {
-      user: ethAddress,
-      amount_buy: amountBuy,
-      token_buy: convertToSignableToken(request.buy),
-      amount_sell: amountSell,
-      token_sell: convertToSignableToken(request.sell),
+  const { ethAddress } = user.imx;
```

Comment:
For the sake of consistency, can we leave the calls to `withPassportError` inside of the workflow files please?
Otherwise we end up in a situation where some workflow methods are wrapped in `withPassportError`, and others aren't.

Reply:
So we want to keep `withDefaultConfirmationScreenTask` outside the workflow file, but the call to `withPassportError` inside it?